site/docs/tools/cli.md

```diff
@@ -34,12 +34,12 @@ The default config file is as follows:
 
 ``` yaml
 auth:
-    # Type can be either basic or aws
-    type: basic
-
-    # Username and password required if using basic auth
```

Comment:
Don't we support "basic" auth in Python anymore?

Reply:
We do, nominally, but servers do not support `BASIC` in `prod`

Reply:
imo we should still document it and just add a note that it's not supported in prod

Reply:
ok, I'll make a follow-up PR for this. Merging this PR to unblock your java client docs changes